src/adn30.f

```diff
@@ -1,12 +1,11 @@
 C> @file
-C> @brief Convert an FXY value from its bit-wise (integer)
-C> representation to its five or six character representation.
+C> @brief Convert an FXY value from an integer
```

Comment:
@jack-woollen or @jbathegit is FXY an acronym? Or what is it? What does it mean?

Reply:
FXY isn't an acronym.  It's a reference notation that's used in the official (WMO) BUFR regulations for Section 3 descriptors, which are normally 6 digits in the form of FXXYYY, where F ranges between 0-3, XX ranges between 0-63, and YYY ranges between 0-255.

Bottom line, the term is well-defined in the official manual.